dashboards/google-cloudsql/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+sample_dashboards:
+  -
+    category: CloudSQL
+    id: 1p-cloudsql-general
+    display_name: CloudSQL General
+    description: |-
+      This dashboard has charts displaying logs from CloudSQL as well as charts of instance states and network throughput.
+    related_integrations:
+      - id: cloudsql
+        platform: GCP
+  -
+    category: CloudSQL
+    id: 1p-cloudsql-replication
+    display_name: CloudSQL Replication
+    description: |-
+      This dashboard has charts displaying information about what replication states and replication performance from CloudSQL.
```

Comment:
information about ~~what~~ replication states?

The sentence doesn't really make sense as is

Reply:
Good spot! Fixed.